src/test/java/test/JmxMetricsTest.java

```diff
@@ -177,7 +177,7 @@ void expose_jmx_metrics_bitbucket_mesh(Product product) throws Exception {
 
         // assert jmx configmap created and has expected config
         final var jmxConfigMap = resources.getConfigMap(product.getHelmReleaseName() + "-jmx-config").getDataByKey("jmx-config.yaml");
-        assertThat(jmxConfigMap).hasTextContaining("- pattern: \".*\"");
+        assertThat(jmxConfigMap).hasTextContaining("- pattern: ");
```

Comment:
Can we check for more?

Reply:
We will, after merging all the products, since some of them use config from the common chart.